    
class BadCredentialsException(Exception):
    def __init__(self):
        self.name = "Error credentials."

